src/nodes.cpp

```diff
@@ -102,6 +102,10 @@ INSTANTIATE_TEMPLATE_METHODS(uint32_t)
 INSTANTIATE_TEMPLATE_METHODS(int64_t)
 INSTANTIATE_TEMPLATE_METHODS(uint64_t)
 
+#ifdef __APPLE__
+INSTANTIATE_TEMPLATE_METHODS(size_t)
+#endif
```

Comment:
I feel like I've done something like this somewhere already…

Reply:
yeah, good old Apple.